vault/logical_system_paths.go

```diff
@@ -393,11 +395,35 @@ func (b *SystemBackend) rekeyPaths() []*framework.Path {
 			Operations: map[logical.Operation]framework.OperationHandler{
 				logical.ReadOperation: &framework.PathOperation{
 					Callback: b.handleRekeyRetrieveBarrier,
-					Summary:  "Return the backup copy of PGP-encrypted unseal keys.",
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
+							Description: "OK",
+							Fields: map[string]*framework.FieldSchema{
+								"nonce": {
+									Type:     framework.TypeString,
+									Required: true,
+								},
+								"keys": {
+									Type:     framework.TypeMap,
+									Required: true,
+								},
+								"keys_base64": {
+									Type:     framework.TypeMap,
+									Required: true,
+								},
+							},
+						}},
+					},
+					Summary: "Return the backup copy of PGP-encrypted unseal keys.",
 				},
 				logical.DeleteOperation: &framework.PathOperation{
 					Callback: b.handleRekeyDeleteBarrier,
-					Summary:  "Delete the backup copy of PGP-encrypted unseal keys.",
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
```

Comment:
```suggestion
						http. StatusNoContent: {{
```
per https://github.com/hashicorp/vault/blob/4b8bd15e28ad3194159e7601fe76a994688988d4/vault/logical_system.go#L909